diff --git a/node_modules/decimal.js/decimal.js b/node_modules/decimal.js/decimal.js
index 23295c6..829a0d7 100644
--- a/node_modules/decimal.js/decimal.js
+++ b/node_modules/decimal.js/decimal.js
@@ -4298,7 +4298,11 @@

       // Retain a reference to this Decimal constructor, and shadow Decimal.prototype.constructor
       // which points to Object.
-      x.constructor = Decimal;
+      Object.defineProperty(x, 'constructor', {
+        configurable: true,
+        value: Decimal,
+        writable: true
+      });

       if (isDecimalInstance(v)) {
         x.s = v.s;
diff --git a/node_modules/decimal.js/decimal.mjs b/node_modules/decimal.js/decimal.mjs
index 71ed02e..5d40433 100644
--- a/node_modules/decimal.js/decimal.mjs
+++ b/node_modules/decimal.js/decimal.mjs
@@ -4293,7 +4293,11 @@ function clone(obj) {

     // Retain a reference to this Decimal constructor, and shadow Decimal.prototype.constructor
     // which points to Object.
-    x.constructor = Decimal;
+    Object.defineProperty(x, 'constructor', {
+      configurable: true,
+      value: Decimal,
+      writable: true
+    });

     if (isDecimalInstance(v)) {
       x.s = v.s;
